tests/test_udf.py

```diff
@@ -380,3 +384,49 @@ def apply_udf_data(data: UdfData):
         mehh_path = Path(data["mehh.__file__"])
         assert not mehh_path.exists()
         assert f"Cleaning up temporary UDF deps at {mehh_path.parent}" in caplog.text
+
+    def test_run_udf_on_vector_data_cube_with_logging(self, tmp_path):
+        repository_root = Path(__file__).parent.parent
+        datacube = openeo.DataCube.load_stac(
+            url=str(repository_root / "docker/local_batch_job/example_stac_catalog/collection.json"),
+            temporal_extent=["2023-06-01", "2023-06-09"],
+        )
+
+        geometries = {
+            "type": "Polygon",
+            "coordinates": [
+                [
+                    [5.07, 51.215],
+                    [5.07, 51.22],
+                    [5.08, 51.22],
+                    [5.08, 51.215],
+                    [5.07, 51.215],
+                ]
+            ],
+        }
+
+        vector_cube = datacube.aggregate_spatial(geometries=geometries, reducer="mean")
+
+        udf = """
+from openeo.udf import UdfData
+import logging
+def apply_udf_data(data: UdfData):
+    logging.warn('Hello from UDF!')
+    return data
+"""
+        post_processed = vector_cube.process("run_udf", data=vector_cube, udf=udf, runtime="Python")
+
+        log_dir = os.environ.get("LOG_DIRS", ".").split(",")[0]
+        log_file = Path(log_dir) / "openeo_python.log"
+        open(log_file, "w").close()  # Clear file content
```

Comment:
how does this work?  there seem to be a lot of brittle assumptions here about `LOG_DIRS` and `openeo_python.log`.  Can't we just make sure that the logging is done to a file under `tmp_path`?

This also looks pretty evil:
```
open(log_file, "w").close() # Clear file content
```

you are blindly clearing a file, possibly/probably in current working directory. That's not something that tests should do

Reply:
I made the message unique, so that the logfile does not need to be cleared first.

I had no direct success with `monkeypatch.setenv("LOG_DIRS", str(tmp_path))`
It does not affect the logging path https://github.com/Open-EO/openeo-python-driver/blob/0e00182d000d052603053632ea5d3c347baf3ba9/openeo_driver/util/logging.py#L88

Reply:
Changed hardcoded path to:
```python
get_logging_config()["handlers"][LOG_HANDLER_FILE_JSON]["filename"]
```
Seems better to me. Ok to merge like this?